src/test/java/com/amihaiemil/zold/mock/MockHttpClientTestCase.java

```diff
@@ -46,24 +47,26 @@
     /**
      * Should return the given response if the request meets the given
      * condition.
-     * 
-     * @throws Exception Unexpected.
      */
     @Test
-    public void returnResponseIfRequestMeetsCondition() throws Exception {
```

Comment:
@wholegroup , could you please share your idea behind this change? Why wrapping the code into `try-catch` and rethrowing the `RuntimeException` is better than adding the `Exception` to the test method signature? The same questions are applicable to other methods of this class.

Reply:
this is a callback and I can't add `Exception` to the test method signature